### ISSUE TITLE
HostKeyCallback is required in ClientConfig

### ISSUE THREAD

**Issue by ajdevries** [bramvdbogaerde/go-scp#3]
Since some security updates in https://godoc.org/golang.org/x/crypto/ssh the `HostKeyCallback` property is required in `ClientConfig`.

So to get this library working, the should be added when building the `ClientConfig`. Also there needs a method/api to add the public host key to the configuration, i.e.
```
clientConfig, _ := auth.PrivateKey("username", "/path/to/rsa/key", "/path/to/hostkey/in/base64format/")

```
- https://github.com/golang/crypto/blob/master/ssh/client.go#L210

**Comment by bramvdbogaerde** [comment]
Thanks for reporting this. I will take a look at it.

**Comment by bramvdbogaerde** [comment]
Fixed by pull request #4
